superdesk/io/feeding_services/ftp.py

```diff
@@ -79,8 +97,14 @@ def _update(self, provider, update):
             except KeyError:
                 logger.warning('missing move_path')
                 move_dest_path = None
+            try:
+                move_dest_path_error = os.path.join(config.get('path', ''), config['move_path_error'])
```

Comment:
the trick is that empty value is used as a way to say "don't move files on error". So we can't use default (or we need an other way to disable on error).
Also the directory needs to exists, if we use default value I guess we need to create the directory, is it a good thing?